estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models
+from datetime import timedelta
+
+class EstatePropertyOffers(models.Model):
+    _name = "estate.property.offers"
```

Comment:
```suggestion
    _name = 'estate.property.offers'
```
`''` for technical terms and `""` for user readable string to be only applied in `py` not `xml`